docs/trace/advanced/stratified-sampling-example/stratified-sampling-example.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
nit: we usually have this inherited automatically for all docs/ projects. So you may skip this.